arch/configure.defaults

```diff
@@ -788,7 +788,7 @@ CC              =       CONFIGURE_CC
 LD              =       $(FC)
 RWORDSIZE       =       CONFIGURE_RWORDSIZE
 PROMOTION       =       #-fdefault-real-8
-ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  CONFIGURE_D_CTSM
+ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  CONFIGURE_D_CTSM  # -DAARCH64_X86_CORRECTNESS_FIX
```

Comment:
It's not at all obvious or documented, but I think adding comments to the end of the `ARCH_LOCAL` lines in configuration stanzas can be problematic. The `ARCH_LOCAL` line is appended by the `Config.pl` script to include NetCDF4 options; see [Config.pl L.801 - 812](https://github.com/wrf-model/WRF/blob/v4.4.1/arch/Config.pl#L801-L812) in the v4.4.1 release.

The result of adding `# -DAARCH64_X86_CORRECTNESS_FIX` to the end of `ARCH_LOCAL` is that anything from `$(NETCDF4_IO_OPTS)` is ignored as a comment. Here's the resulting line in my `configure.wrf` file:
```
ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  -DWRF_USE_CLM  # -DAARCH64_X86_CORRECTNESS_FIX $(NETCDF4_IO_OPTS)
```
One possible solution might be to just provide an alternative `ARCH_LOCAL` definition as a comment:
```
ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  CONFIGURE_D_CTSM
# ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  CONFIGURE_D_CTSM -DAARCH64_X86_CORRECTNESS_FIX
```
Longer-term, I think the best solution would be to fix the logic in the `Config.pl` script, and I'd be willing to help with this, but it might be a little while before I could do so.

Reply:
Michael,

I am not aware of the Config.pl.  Thanks for pointing this out.  The second solution works for me.  It would be nice that you can help with the logic in Config.pl; but it is not required at the moment.

Regards,

Jun